omigami/omigami
# OMI-413: Refactor molecule plot

https://datarevenue.atlassian.net/browse/OMI-413

Changed function, test and updated spec2vec tutorial

TODO:
- Check out an image (test visually)
- Update ms2ds notebook
